index.html

```diff
@@ -1,111 +1,424 @@
+                </div>
+                <article>
+                  <h1>Sonic live</h1>
+                  <p>
+                    One September 22, 2021 A volutpat nulla eros, nulla ultrices
+                    nec nulla imperdict
+                  </p>
+                </article>
+                <button>Read more ></button>
+              </div>
             </div>
-
-            <div class="body">
-              <div class="profile-icon-box">
-                <div class="profile-icon">
-                    <img src="https://assets.codepen.io/4292193/ayakoyokoya_icon2.png" />
+            <div class="mini-post">
+              <div>
+                <img src="#" alt="text" />
+              </div>
+              <div>
+                <div>
+                  <p><span class="folder icon"></span>Tony Nguyen</p>
+                  <p><span class="folder icon"></span>20 Jun 2022</p>
                 </div>
+                <article>
+                  <h1>Sonic live</h1>
+                  <p>
+                    One September 22, 2021 A volutpat nulla eros, nulla ultrices
+                    nec nulla imperdict
+                  </p>
+                </article>
+                <button>Read more ></button>
               </div>
-
-              <div class="name">
-                <p>error!!!!</p>
-                <p>error!!!!</p>
-                <p>error!!!!</p>
-                <p>error!!!!</p>
-                <p>error!!!!</p>
-                <button class="ok-error-btn" onclick="window.location.reload();">OK...</button>
+            </div>
+          </div>
+          <div class="post">
+            <div>
+              <img src="#" alt="text" />
+            </div>
+            <div>
+              <div>
+                <p><span class="folder icon"></span>Tony Nguyen</p>
+                <p><span class="folder icon"></span>20 Jun 2022</p>
               </div>
+              <article>
+                <h1>Sonic live</h1>
+                <p>
+                  One September 22, 2021 A volutpat nulla eros, nulla ultrices
+                  nec nulla imperdict
+                </p>
+              </article>
+              <button>Read more ></button>
             </div>
           </div>
         </div>
-      </template>
-      <!-- <script src="script.js"></script> --> <!-- uncomment script --> 
-</body>
+        <button>Read more</button>
+      </section>
+      <section class="about-as">
+        <div>
+          <div>
+            <h1>About us</h1>
+            <p>
+              Duls aute irure dolor in reprehenderit in voluptate velit esse
+              cillum dolore eu fugiat nulla pariatur. Excepteur sint occae cat
+              cupidatat not prodient, sunt in culpa qui officia dese runt molit
+              anim id est laborum velit esse cillium dolore eu fugiat nulla
+              pariatu epteur sint occaecat.
+            </p>
+          </div>
+          <div>
+            <h1>High quality</h1>
```

Comment:
The 'h1' tag should be used for the main title of the page. This is typically the highest level heading and should only be used once per page. For this case it is better to use the 'h3' tag